Cargo.toml

```diff
@@ -81,7 +81,7 @@ kube = { version = "0.98", default-features = false, features = [
     "rustls-tls",
     "client",
 ] }
-k8s-openapi = { version = "0.24.0", features = ["v1_28"] }
+k8s-openapi = { version = "0.24.0", features = ["v1_32"] }
```

Comment:
do we still need this in lakekeeper?

Reply:
Yes - won't compile otherwise as the k8s dependency pulled in by limes doesn't know which k8s API Version to build for.
Good point actually - let me try to only depend on it in the bin crate.

Reply:
Actually no, because we fixed it in limes.
I think removing it there should be done in a later version. Removing it from lakekeeper for now